cypress/support/generate.js

```diff
@@ -0,0 +1,11 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const username = faker.internet.userName();
+  const email = `${username}@mail.com`;
+  const password = '12345Qwert!';
```

Comment:
Hardcoding the password is not a good practice, even in test data generation. It would be better to generate a random password using faker. For example, you can use `faker.internet.password();` to generate a random password.